Jenkinsfile

```diff
@@ -1,10 +1,16 @@
 elifePipeline {
     node('containers-jenkins-plugin') {
         def commit
+        def fullImageTag
 
         stage 'Checkout', {
             checkout scm
             commit = elifeGitRevision()
+            fullImageTag = sh(
+                script: "docker-compose config | grep -P -o '(?<=\\simage: elifesciences/sciencebeam-trainer-grobid-builder:)\\S+'",
```

Comment:
grepping YAML is sketchy

I suggest to pass in the classic `IMAGE_TAG=${commit}` at build time; then you can retrieve it, tag it as you like and push a more verbose tag:
```
def image = DockerImage.elifesciences(this, 'sciencebeam-trainer-grobid', commit)
def grobidTag = sh "docker-read-label ${image}", returnStdout: true`
def fullImageTag = "${grobidTag}-${commit}"
def unstable_image = image.addSuffixAndTag('_unstable', fullImageTag)
unstable_image.push()
```

Reply:
I agree that it's sketchy.

I think it might be confusing if the non-pushed tag isn't matching the pushed tag. It could also cause issues if we for example ran two builds at the same time from the same commit but different grobid versions. (or at least the same tag could refer to different builds)

Would getting the grobid version from `.env` be easier?

Reply:
`sh 'source .env && echo $GROBID_TAG`
But it does not make sense for the case of two builds of two different grobid versions, if there's only one version in `.env`.

Reply:
Yes, right now there is only one version. But I was thinking of having it build for at least two versions. Of course it wouldn't be in `.env` in that case. Where would you put it? (but then that would be a separate PR)

Reply:
Wherever it's easier, with the caveat that it would probably be expressed as an environment variable passed to `docker-compose build` (overrides `.env`) so its retrieval should keep with that in mind (able to read environment variables or to read the values first from the same place that is read from the Jenkinsfile to create them).